js/web/pages/index.js

```diff
@@ -16,7 +16,7 @@ export default function Home() {
       }
       router.push(path)
     }
-  }, [wallet])
+  }, [wallet, router])
```

Comment:
lets remove this as i dont think we were doing something intentional w the change